Runtime/Game/Requests/NotificationRequests.cs

```diff
@@ -0,0 +1,468 @@
+﻿using System.Collections.Generic;
+using LootLocker.LootLockerEnums;
+
+namespace LootLocker.LootLockerEnums
+{
+    /// <summary>
+    /// Enum of the different available priorities for notifications
+    /// </summary>
+    public enum LootLockerNotificationPriority
+    {
+        low = 0,
+        medium = 1,
+        high = 2,
+        emergency = 3,
+    };
+
+    /// <summary>
+    /// Enum of the different kinds of notification bodies possible, use this to figure out how to parse the notification body
+    /// </summary>
+    public enum LootLockerNotificationContentKind
+    {
+        group_reward = 0,
+        currency_reward = 1,
+        asset_reward = 2,
+        progression_reset_reward = 3,
+        progression_points_reward = 4,
+    };
+}
+
+namespace LootLocker.Requests
+{
+    //==================================================
+    // Data Definitions
+    //==================================================
+
+    /// <summary>
+    /// </summary>
+    public class LootLockerNotificationContextEntry
+    {
+        /// <summary>
+        /// The key for this context entry
+        /// </summary>
+        public string Key { get; set; }
+        /// <summary>
+        /// The value of this context entry
+        /// </summary>
+        public string Value { get; set; }
+    };
+
+    /// <summary>
+    /// </summary>
+    public class LootLockerNotificationRewardCurrency
+    {
+        /// <summary>
+        /// The date the Currency reward was created.
+        /// </summary>
+        public string Created_at { get; set; }
+        /// <summary>
+        /// The date the Currency reward was last updated.
+        /// </summary>
+        public string Updated_at { get; set; }
```

Comment:
Change created_at and updated_at to Date instead, easier to handle.

Reply:
Fixed